client/matchmaking/find-match.jsx

```diff
@@ -186,7 +189,7 @@ class Find1vs1MatchForm extends React.Component {
             <BrowseButton onClick={onBrowsePreferred}>
               <RandomContainer>
                 <RandomIcon />
-                <SubheadingOld>Random map</SubheadingOld>
+                <Subtitle1>Random map</Subtitle1>
```

Comment:
This should be `Subtitle2`. General rule is, things that are "ShieldBattery content" get put in Roboto Condensed:
users, maps, etc.

Refer to the change I made to matchmaking map pool screen and make this match